frappe/website/doctype/personal_data_deletion_request/personal_data_deletion_request.py

```diff
@@ -330,14 +351,28 @@ def remove_unverified_record():
 		AND `creation` < (NOW() - INTERVAL '7' DAY)"""
 	)
 
+def update_sla():
+	account_deletion_sla = frappe.db.get_single_value("Website Settings", "account_deletion_sla")
+	if account_deletion_sla < 1:
+		return
+
+	requests = frappe.get_all("Personal Data Deletion Request",
+				filters = {
+					"sla_status": "Open"
+				},
+				fields = ["name", "creation", "status"])
```

Comment:
```suggestion
	requests = frappe.get_all("Personal Data Deletion Request",
		  filters = {
			  "sla_status": "Open"
		  },
		  fields = ["name", "creation", "status"])
```
```suggestion
	requests = frappe.get_all("Personal Data Deletion Request",
				filters = {
					"sla_status": "Open"
				},
				fields = ["name", "creation", "status"])
```

Reply:
@surajshetty3416 Isn't the second suggestion the same as what is already present?

Reply:
Seems like an issue with GitHub... it added the review twice.